conf/messages

```diff
@@ -30,9 +30,9 @@ org.details.hint.notllp=Corporation Tax, including your CT600 form
 org.details.hint=This is 10 numbers, for example 1234567890. It will be on tax returns and other letters about {0}. It may be called ‘reference’, ‘UTR’ or ‘official use’. You can <a href="https://www.gov.uk/find-lost-utr-number">find a lost UTR number</a>.
 
 tradingdetails.fsrref.label=Financial Conduct Authority reference
-tradingdetails.fsrref.hint=This is 6 numbers.
+tradingdetails.fsrref.hint=This is 6 numbers, like 123456
```

Comment:
rebase with master as this changes are already available in master

Reply:
Done Pratik. Thanks